packages/detox/src/generators/application/application.ts

```diff
@@ -38,6 +43,26 @@ export async function detoxApplicationGeneratorInternal(
   const lintingTask = await addLinting(host, options);
   const depsTask = ensureDependencies(host, options);
 
+  updateTsconfigFiles(
+    host,
+    options.e2eProjectRoot,
+    'tsconfig.json',
+    {
+      outDir: 'out-tsc/detox',
+      allowJs: true,
+      types: ['node', 'jest', 'detox'],
```

Comment:
We need to add the `module`/`moduleResolution` expected options for Detox.

Reply:
Done